src/tabs/receiver.html

```diff
@@ -314,7 +314,7 @@
                                 </div>
                             </td>
                             <td>
-                                <div class="helpicon cf_tip" i18n_title="receiverRcSmoothingAutoFactorHelp"></div>
+                                <div class="helpicon cf_tip receiverRcSmoothingAutoFactorHelp" i18n_title="receiverRcSmoothingAutoFactorHelp"></div>
```

Comment:
@haslinghuis I think you are getting that error because of the `i18n_title="receiverRcSmoothingAutoFactorHelp"` attribute, try removing it and it should go away.